ignite/metrics/gan/fid.py

```diff
@@ -12,6 +12,12 @@
 ]
 
 
+if Version(torch.__version__) <= Version("1.7.0"):
+    torch_outer = torch.ger
```

Comment:
Is codecov warning OK?

Reply:
Yes, we do not run the CI on older pth that's why we do not enter it here. I can also run manually https://github.com/pytorch/ignite/actions/workflows/pytorch-version-tests.yml to ensure that everything is good.